usaspending_api/download/sql/award_financial.sql

```diff
@@ -0,0 +1,12 @@
+DROP VIEW IF EXISTS vw_award_financial;
+CREATE VIEW vw_award_financial AS
+SELECT
+    "financial_accounts_by_awards".*,
+	"cgac_aid"."agency_name" AS "agency_identifier_name",
+	"cgac_ata"."agency_name" AS "allocation_transfer_agency_identifier_name"
+FROM "financial_accounts_by_awards"
+INNER JOIN "submission_attributes" ON ("financial_accounts_by_awards"."submission_id" = "submission_attributes"."submission_id")
+LEFT OUTER JOIN "treasury_appropriation_account" ON ("financial_accounts_by_awards"."treasury_account_id" = "treasury_appropriation_account"."treasury_account_identifier")
+LEFT OUTER JOIN "cgac" AS "cgac_aid" on ("treasury_appropriation_account"."agency_id" = "cgac_aid"."cgac_code")
+LEFT OUTER JOIN "cgac" AS "cgac_ata" on ("treasury_appropriation_account"."allocation_transfer_agency_id" = "cgac_ata"."cgac_code")
```

Comment:
I guess i'm thinking -- could there be more tucked into this sql to take some of the work off the Django ORM and make the download queries even faster? 

I see now that this code was all aimed at fixing that `cgac` join, but this does provide a good clean foundation for continuing to chip away at the ugly ORM-produced-sql, and instead put cleaner and more efficient starter-sql in here.

Reply:
After seeing the SQL though, i see these don't go too deep into download-specific sql (e.g. joining awards and such). Should they? 
If they will head there, i'd say suffix with `downloads` or `dl`. If they will stay a minimally augmented version of the file A / B / C tables, then maybe leave them named as is.

Reply:
Another thought that comes to mind if it was just to augment the data with `cgac` fields, could we have just done that at ETL-time, when pulling this data over from broker (join in cgac where it exists, and update these tables as that data changes?)

Reply:
> I guess i'm thinking -- could there be more tucked into this sql to take some of the work off the Django ORM and make the download queries even faster?

Was hoping to add more as well. Minimal "extra" fields on the View will help to reduce data changes for downloads in the future (except in terrible performance scenarios). Additionally, the other parts that aren't a straight `SELECT` (e.g. `CASE` statements) are easily done with Django or they need to be dynamically added based on filters (e.g. the `gross_outlay_amount_by_award_cpe` field).

> If they will head there, i'd say suffix with downloads or dl

Good call, will add that to the naming convention. 

>Another thought that comes to mind if it was just to augment the data with cgac fields, could we have just done that at ETL-time, when pulling this data over from broker (join in cgac where it exists, and update these tables as that data changes?)

Another possible route. Personally more of a fan for trying to keep some USAspending tables closer to their Broker counter part.